src/main/java/org/folio/consortia/service/impl/UserAffiliationServiceImpl.java

```diff
@@ -62,4 +62,22 @@ public void createPrimaryUserAffiliation(String eventPayload) {
       log.error("Exception occurred while creating primary affiliation", e);
     }
   }
+
+  @Override
+  @SneakyThrows
+  public void deletePrimaryUserAffiliation(String data) {
+    try {
+      var userEvent = OBJECT_MAPPER.readValue(data, org.folio.consortia.domain.dto.UserEvent.class);
```

Comment:
please use our logging reccomendation for new code as well, for example here we need to add log that we received event for particular userId + tenantId (these 2 params can identify user event entity)
Also please add userId + tenantId to other existing logs and to addPrimaryAffiliation method

Reply:
Thanks, I added